project/Build.scala

```diff
@@ -789,7 +818,7 @@ object Build {
 
   val commonLinkerInterfaceSettings = Def.settings(
       commonSettings,
-      publishSettings,
+      publishSettings(VersionScheme.BreakOnSevere),
```

Comment:
I think this one should be fine. We create the dependency on `scalajs-linker` with a version number that is dynamically looked up from `ScalaJSVersions.current`. If there is a newer version of `linker-interface`, then there will be a newer version of `scalajs-ir`, which will in turn cause the newer version of `scalajs-linker` to be selected.

We could still end up in a scenario where scalajs-ir is bumped higher than linker-interface. But that is an illegal scenario to begin with, since scalajs-ir breaks on Minor, so having a dependency from linker-interface 1.x to scalajs-ir 1.x evicted in favor of scalajs-ir 1.y is already reported as an issue.